students/carlos_novoa/lesson01/break_me.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+"""
+Lesson1, Task 1: Explore Errors
+"""
+
+
+def name_error():
+    print('abc'+1)
```

Comment:
This function will cause a `TypeError` rather than a `NameError`. In Python, you don't have to name values before performing operations with them. So the error here is from trying to add/concatenate a string and integer (`TypeError`) rather than using a symbol that has not been assigned a value (`NameError`).

Reply:
I don't know how I overlooked that. Revised to:

`def name_error():
    print(error)`